lib/matplotlib/__init__.py

```diff
@@ -578,7 +558,21 @@ def checkdep_usetex(s):
     return flag
 
 
-def _get_home():
+def _log_result(fmt, level="INFO", func=None):
```

Comment:
If we are going to use this, it could use a comment along the lines of the explanation in the comments above.  I understand it now, sort of, but would never have understood it if it wasn't explained, just due to an unfamiliarity w/ functools. OTOH, I can follow @efiring code.  I *think* the advantage here over @efiring is that you've made it so all the different calls don't need to have the same logic to avoid duplicate system calls.  Does that make the name a bit of a misnomer?  The real purpose is the wrapping, isn't it?